### PR TITLE
Studio: Update Studio scheme protocol handler to AppxManifest for Windows

### PR DESCRIPTION
**Pull request by gavande1**
## Related issues

- Fixes STU-936

## Proposed Changes

- Update Studio protocol from  from wpcom-local-dev to wp-studio to the existing app manifest

## Testing Instructions

Taken from https://github.com/Automattic/studio/pull/1969, Props to @bcotrim 

1. Download the signed .appx file from the CI build artifacts for this PR (https://buildkite.com/automattic/studio/builds/9170/steps/canvas?jid=019aa14c-6f7d-4be5-bbac-d950b13e7e91)
2. Install the .appx on a Windows machine (right click -> Install)
3. Open Studio and click the "Login" button in the top right
4. Authenticate in the browser when redirected to WordPress.com
5. After approving, verify the browser shows "Authorized. You can now open Studio and close this window."
6. Verify Studio successfully logs you in and displays your account information

## Pre-merge Checklist

- [ ] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

**Comment by gavande1** [comment]
This needs to be deployed after https://github.com/Automattic/studio/pull/2029 is deployed.

File: common/constants.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const DAY_MS = HOUR_MS * 24;
 
 // OAuth constants
 export const CLIENT_ID = '95109';
-export const PROTOCOL_PREFIX = 'wpcom-local-dev';
+export const PROTOCOL_PREFIX = 'wp-studio';
```
Comment (gavande1) [inline common/constants.ts:9]: This is a temporary change until https://github.com/Automattic/studio/pull/2029 is merged.

**Comment by gcsecsey** [comment]
In https://github.com/Automattic/studio/pull/2018 we made builds on PRs optional, so I started the Windows builds on Buildkite for this PR. ⌛

**Comment by gavande1** [comment]
Thanks @gcsecsey for running the Windows build.

**Review by gcsecsey — APPROVED** [review]
Thanks for updating the handler @gavande1! This works well for me on Windows. 👍 

https://github.com/user-attachments/assets/caea78e8-f462-486f-899f-fc6ddb603250

**Comment by gavande1** [comment]
I am deploying this for beta release. Feel free to test it again for edge cases.
